libs/components/src/lib/accordion-item/accordion-item.spec.ts

```diff
@@ -0,0 +1,117 @@
+import { elementUpdated, fixture, getControlElement } from '@vivid-nx/shared';
+import { AccordionItem } from './accordion-item';
+import '.';
+
+const COMPONENT_TAG = 'vwc-accordion-item';
+
+describe('vwc-accordion-item', () => {
+	let element: AccordionItem;
+
+	beforeEach(async () => {
+		element = (await fixture(
+			`<${COMPONENT_TAG}></${COMPONENT_TAG}>`
+		)) as AccordionItem;
+	});
+
+	describe('basic', () => {
+		it('should be initialized as a vwc-accordion-item', async () => {
+			expect(element).toBeInstanceOf(AccordionItem);
+			expect(element.open).toBeFalsy();
+			expect(element.dense).toBeFalsy();
+			expect(element.icon).toBeUndefined();
+			expect(element.iconTrailing).toBeFalsy();
+			expect(element.meta).toEqual('');
+			expect(element.noIndicator).toBeFalsy();
+			expect(element.heading).toEqual('');
+			expect(element.headingLevel).toBeUndefined();
+		});
+	});
+
+	describe('show', () => {
+		it('should set "open" to true and add "open" class', async () => {
+			const control = getControlElement(element);
+			const hasClassOpenBeforeShow = control.classList.contains('open');
+
+			element.show();
+			await elementUpdated(element);
+			const hasClassOpenAfterShow = control.classList.contains('open');
+
+			expect(element.open).toEqual(true);
+			expect(hasClassOpenBeforeShow).toEqual(false);
+			expect(hasClassOpenAfterShow).toEqual(true);
+		});
+	});
+
+	describe('hide', () => {
+		it('should set "open" to false and remove "open" class', async () => {
```

Comment:
I think we need to keep definition as simple as possible
```suggestion
		it('should unset "open"', async () => {
```